test/attr-for-selected.html

```diff
@@ -197,6 +197,18 @@
         assert.deepEqual(selector.selectedValues, ['non-existing-value', 'value0']);
         assert.deepEqual(selector.selectedItems, [items[0]]);
       });
+
+      test('default element not found does not result in infinite loop', function(done) {
+        selector.fallbackSelection = 'non-existing-fallback';
+        selector.select('non-existing-value');
+        assert.equal(selector.selectedItem, undefined);
+        selector.multi = true;
+        selector.selectedValues = ['non-existing-value'];
+        assert.deepEqual(selector.selectedItems, [undefined]);
+        selector.fallbackSelection = 'default';
+        assert.deepEqual(selector.selectedItems, [items[2]])
+        done();
```

Comment:
Oh, I didn't consider that but given that the test is meant to trigger an infinite loop on failure, when it fails in that way the whole test suite will time out! :)